zboxcore/sdk/downloadworker.go

```diff
@@ -224,6 +224,21 @@ func (req *DownloadRequest) processDownload(ctx context.Context) {
 		return
 	}
 
+	// the smallest ChunkSize is 64kb
```

Comment:
it is not true. default is 64kb. but 1kb is fine too. Because chunk_size is customizable now.